### PR TITLE
LPS-156259 Block groovy script creation on LXC

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-156259

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:reevaluate:1387401_2555

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/ObjectActionExecutorRegistryImpl.java [hunk 1]
```diff
@@ -90,7 +92,18 @@ protected void activate(BundleContext bundleContext) {
...
+
+					if (!(objectActionExecutor instanceof
+							GroovyObjectActionExecutorImpl)) {
+
+						emitter.emit(objectActionExecutor.getKey());
+					}
+				}
+				else {
+					emitter.emit(objectActionExecutor.getKey());
+				}
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/ObjectActionExecutorRegistryImpl.java:106]: @mateussgn can you try this:

```
if (GetterUtil.getBoolean(
        PropsUtil.get("block-groovy-scripts")) && (objectActionExecutor instanceof
            GroovyObjectActionExecutorImpl)) {

    return;
}

emitter.emit(objectActionExecutor.getKey());
```

I think it's clearer

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/ObjectValidationRuleEngineTrackerImpl.java [hunk 2]
```diff
@@ -63,7 +65,18 @@ public void map(
...
+
+						if (!(objectValidationRuleEngine instanceof
+								GroovyObjectValidationRuleEngineImpl)) {
+
+							emitter.emit(objectValidationRuleEngine.getName());
+						}
+					}
+					else {
+						emitter.emit(objectValidationRuleEngine.getName());
+					}
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/ObjectValidationRuleEngineTrackerImpl.java:79]: @mateussgn same here

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by mateussgn** [comment]
ci:test:relevant

**Review by rodrigopaulino — APPROVED** [review]
LGTM! 🎉

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:forward

**Pull request closed without merging**